stix_shifter_modules/cybereason/stix_transmission/ping_connector.py

```diff
@@ -0,0 +1,35 @@
+from stix_shifter_utils.modules.base.stix_transmission.base_ping_connector import BasePingConnector
+from stix_shifter_utils.utils.error_response import ErrorResponder
+from stix_shifter_utils.utils import logger
+from json.decoder import JSONDecodeError
+import json
+
+
+class PingConnector(BasePingConnector):
+    def __init__(self, api_client):
+        self.api_client = api_client
+        self.logger = logger.set_logger(__name__)
+
+    def ping_connection(self):
+        """
+        Ping the endpoint
+        :return: dict
+        """
+        return_obj = {}
+        response_dict = {}
+        try:
+            response = self.api_client.ping_box()
+            response_code = response.response.status_code
+            response_dict = json.loads(response.response.text)
+            if response_code == 200 and response_dict['status'] == 'SUCCESS':
+                return_obj['success'] = True
+        except JSONDecodeError:
+            response_dict['type'] = "AuthenticationError"
+            response_dict['message'] = "Invalid Authentication"
+            ErrorResponder.fill_error(return_obj, response_dict, ['message'])
+        except Exception as ex:
+            response_dict['type'] = ex.__class__.__name__
+            response_dict['message'] = ex
+            self.logger.error('error while pinging: %s', ex)
+            ErrorResponder.fill_error(return_obj, response_dict, ['message'])
+        return return_obj
```

Comment:
I tested with this ping CLI with invalid hostname which returns error but with `unknown` code. It should have a valid error code-
`python main.py transmit cybereason '{"host":"test.com", "port": 443}'  '{ "auth": { "username": "xxx", "password": "xxx" } }'  ping`

Response:
`{
    "success": false,
    "error": "HTTPSConnectionPool(host='test.com', port=443): Max retries exceeded with url: /login.html (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ff20d21ada0>, 'Connection to test.com timed out. (connect timeout=2)'))",
    "code": "unknown"
}`

Reply:
Code changes has been done and 'ConnectionError'  will be thrown in connector.py file.